deploy/backplane/10-cee-read.ClusterRole.yml

```
@@ -0,0 +1,18 @@
+aggregationRule:
+  clusterRoleSelectors:
+    # aggregate all "view" scope rbac
+    - matchExpressions:
+        - key: rbac.authorization.k8s.io/aggregate-to-view
+          operator: In
+          values:
+            - "true"
+    - matchExpressions:
+        - key: managed.openshift.io/aggregate-to-backplane
+          operator: In
+          values:
+            - "true"
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: backplane-cee-read-all
```

```suggestion
  name: backplane-cee
```
Why "read" and "all" in the name?

---

was trying to be descriptive when I first created the role with its name, but it is not consistent. I will make remove '-read-all'